packages/core/src/components/cv-toggle/cv-toggle-skeleton.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="cv-toggle bx--form-item">
+    <input disabled type="checkbox" class="bx--toggle bx--skeleton" :id="uid" />
```

Comment:
The react version omits the following disabled attribute and the id attribute. As a result the input cannot be clicked.

An alternative if we wish to keep the id attribute is to use '@click.prevent' on the input. As a result the colors match the react component.